src/eftest/runner.clj

```diff
@@ -84,10 +84,14 @@
   ([vars opts]
    (let [start-time (System/currentTimeMillis)]
      (if (empty? vars)
-       (println "No tests found.")
+       (do
+         (println "No tests found.")
+         test/*initial-report-counters*)
        (binding [report/*context* (atom {})
                  test/report      (:report opts progress/report)]
          (test/do-report {:type :begin-test-run, :count (count vars)})
          (let [counters (test-all vars opts)
-               duration (- (System/currentTimeMillis) start-time)]
-           (test/do-report (assoc counters :type :summary, :duration duration))))))))
+               duration (- (System/currentTimeMillis) start-time)
+               summary (assoc counters :type :summary, :duration duration)]
```

Comment:
I didn't add `:type :summary`, and it appears that's what [`clojure.test` does too](https://github.com/clojure/clojure/blob/2e0c0a9a89ede8221504edeb90e8c4ee6cce7e16/src/clj/clojure/test.clj#L767-L777).

Do you want to `dissoc` the `:type` before returning the map?